devise-security.gemspec

```diff
@@ -46,7 +41,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'pry-rescue'
   s.add_development_dependency 'rails-controller-testing'
   s.add_development_dependency 'rails_email_validator'
-  s.add_development_dependency 'rubocop', '~> 0.80.0' # NOTE: also update .codeclimate.yml and make sure it uses the same version
+  s.add_development_dependency 'rubocop', '~> 0.83.0' # NOTE: also update .codeclimate.yml and make sure it uses the same version
```

Comment:
It looks like this was done 👍 

https://github.com/devise-security/devise-security/pull/183/files#diff-0655d430acfa953cae08b6619689d18cR46